pkgs/applications/system/monitor/default.nix

```diff
@@ -53,8 +56,15 @@ stdenv.mkDerivation rec {
     libwnck
     sassc
     udisks2
+    libX11
+    libXext
+    libXNVCtrl
   ];
 
+  NIX_LDFLAGS = "-lXext";
```

Comment:
I consider this as a dirty hack, maybe we can document why we do this.

Reply:
yes. can you phrase a comment? i don't know what it does